### DISCUSSION TITLE
第一次同步成功了，第二次新建一个帖子后又报错了

### DISCUSSION THREAD

**Discussion by Lyman11** [Xheldon/Notion-Flow-Prod#5] [Q&A]
❌ [Notion] Notion pages.retrieve 接口报错:
TypeError: Cannot read properties of undefined (reading 'length')
❌ [Notion] Notion 内容处理过程中出现错误，请检查日志:

**Comment by Lyman11** [comment]
后面删了帖子，也不行，重新拷贝了你的模板过来也不行。。。

**Reply by Xheldon** [reply]
如果成功过，说明首次配置没问题，这个错误看起来似乎是你的内容异常或者读取内容失败。
1. 首先检查下你的梯子是否稳定先。
2. 这个报错的地方代码如下：
![CleanShot_2024_10_30_at_01_45_48@2x](https://github.com/user-attachments/assets/95b03b9a-2f9c-4745-a57d-b728e5e07ad2)
3. 严格按照文档要求的那几个必要字段（目前看应该是你必填属性未填导致的），不要缺，title 那几个，你对照文档检查下看看。
